src/widgets/CustomerRequisitionItemDetail.js

```diff
@@ -14,12 +14,12 @@ import { getPageInfoColumns } from '../pages/dataTableUtilities';
 
 import { DARKER_GREY, WHITE, SUSSOL_ORANGE } from '../globalStyles';
 
-const RequisitionItemDetailsComponent = ({ item }) => (
+const CustomerRequisitionItemDetailsComponent = ({ item }) => (
   <View style={localStyles.container}>
     <PageInfo
       titleColor={SUSSOL_ORANGE}
       infoColor={WHITE}
-      columns={getPageInfoColumns('requisitionItemDetail')(item)}
+      columns={getPageInfoColumns('customerRequisitionItemDetail')(item)}
```

Comment:
Diff is a bit strange because I just copy pasted this component, and changed this string. This is implemented as different components because of premature optimization, I guess. Essentially predicting that these 'row detail' components will probably become used more heavily, at some point. But I bet because they're not all a single component with conditionals it won't be!